hooks/adoclink-check.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+# OSX GUI apps do not pick up environment variables the same way as Terminal apps and there are no easy solutions,
+# especially as Apple changes the GUI app behavior every release (see https://stackoverflow.com/q/135688/483528). As a
+# workaround to allow GitHub Desktop to work, add this (hopefully harmless) setting here.
+export PATH=$PATH:/usr/local/bin
+
+if ! command -v asciidoc-link-check; then
```

Comment:
Is this related to this PR?

Reply:
Doh! This is a WIP script that I did not intend to check in. Removed! 96f2c4f